book/src/validator-manager-create.md

```diff
@@ -0,0 +1,175 @@
+# Creating and Importing Validators
+
+[Ethereum Staking Launchpad]: https://launchpad.ethereum.org/en/
+
+The `lighthouse validator-manager create` command derives validators from a
+mnemonic and produces two files:
+
+- `validators.json`: the keystores and passwords for the newly generated
+    validators, in JSON format.
+- `deposits.json`: a JSON file of the same format as
+    [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli) which can
+    be used for deposit submission via the [Ethereum Staking
+    Launchpad][].
+
+The `lighthouse validator-manager import` command accepts a `validators.json`
+file (from the `create` command) and submits those validators to a running
+Lighthouse Validator Client via the HTTP API.
+
+These two commands enable a workflow of:
+
+1. Creating the validators via the `create` command.
+1. Importing the validators via the `import` command.
+1. Depositing validators via the [Ethereum Staking
+    Launchpad][].
+
+The separation of the `create` and `import` commands allows for running the
+`create` command on an air-gapped host whilst performing the `import` command on
+an internet-connected host.
+
+The `create` and `import` commands are recommended for advanced users who are
+familiar with command line tools and the practicalities of managing sensitive
+cryptographic material. **We recommend that novice users follow the workflow on
+[Ethereum Staking Launchpad][] rather than using the `create` and `import`
+commands.**
+
+## Simple Example
+
+Create validators from a mnemonic with:
+
+```bash
+lighthouse \
+    validator-manager \
+    create \
+    --first-index 0 \
```

Comment:
I agree - although I wasn't sure if it was left out intentionally to prevent users from entering an network other than mainnet. Selecting the right network is probably one of the important bits in key generation - selecting an incorrect one will result in incorrect signature (due to a different `genesis_fork_version` used) which makes the deposit invalid. `validator-manager` already checks that the BN is on the same network though, so we're good. 

Interesting fact: there are 10 invalid deposits on mainnet and 837 invalid deposits on goerli! Most of them probably are from custom solutions to make deposits rather than using the launchpad, which does validation on the signature and network.